app/models/taxon.rb

```diff
@@ -1129,6 +1132,23 @@ def taxon_cant_be_its_own_ancestor
     errors.add( :base, "can't be its own ancestor" )
   end
 
+  def provisional_only_for_fungi
+    return unless provisional
+    return if new_record? && provisional_was.nil? # allow for new records
```

Comment:
What is `provisional_was.nil?` doing here? I don't think it will ever be nil because the default is `false`. Can this just check for `new_record?`